cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -342,8 +347,14 @@ public void pushPackage(Path balaPath, String org, String name, String version,
         }
     }
 
+    /**
+     * Pulling a package.
```

Comment:
Shall we refactor the method comment to use proper tense and add the parameters and exceptions thrown?

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-lang/commit/a26d8a97e757d634eba17907656536dfe134e2d5